lib/net/ldap.rb

```diff
@@ -1323,4 +1317,17 @@ def new_connection
     }
     raise e
   end
+
+  # Normalize encryption parameter the constructor accepts, expands a few
+  # convenience symbols into recognizable hashes
+  def normalize_encryption(args)
+    return if args.nil?
+    return args if args.is_a? Hash
+
+    case method = args.to_sym
+    when :simple_tls, :start_tls
+      { :method => method, :tls_options => {} }
+    end
+  end
```

Comment:
ah, sorry, I misunderstood. the purpose of this method is to normalize (I was thinking of validating a given `encryption` option when I saw your PR). 
Anyway, please forget about my comment :bow: